index.js

```
@@ -48,6 +48,44 @@
+// Get default branch name
+export function getDefaultBranch() {
+  try {
+    // Try to get the remote's default branch
+    // First get the default remote (usually origin)
+    const remote = execSync('git remote').toString().trim().split('\n')[0];
+    
+    // Then get the default branch (what HEAD points to)
+    const output = execSync(`git remote show ${remote} | grep "HEAD branch"`).toString().trim();
+    const match = output.match(/HEAD branch:\s*(.+)$/);
+    
+    if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)